docs/source/upgrade.rst

```diff
@@ -609,13 +665,13 @@ Compatibility Matrix
      - Docs
    * - v14
      - 3.5~
-     - 11.2~
-     - 4.3~
+     - 12.0~
+     - 7.0~
      - 2.0~
-     - 2.16~
+     - 2.18~
      - n/a
      - 3.10~
-     - 1.22~
+     - 2.0~
      - 1.7~
      - NumPy 1.26 & SciPy 1.11
```

Comment:
@kmaehashi is this correct?
```suggestion
     - NumPy 2.0 & SciPy 1.11
```

Reply:
We use NumPy v2.3 and SciPy v1.16 as baseline.

https://docs.cupy.dev/en/latest/install.html#python-dependencies

```suggestion
     - NumPy 2.3 & SciPy 1.16
```